apps/widget-configurator/src/app/configurator/index.tsx

```diff
@@ -47,6 +48,48 @@ const DEFAULT_STATE = {
   buyAmount: 0,
 }
 
+const COW_LISTENERS: CowEventListeners = [
+  {
+    event: CowEvents.ON_TOAST_MESSAGE,
+    handler: (event) => {
+      console.log('[configurator] 🍞 Toast message', event.message, event.data)
+
+      // You can provide a simplistic way to handle toast messages (use the "message" to show it in your app)
+      if (event.messageType === ToastMessageType.SWAP_ETH_FLOW_SENT_TX) {
```

Comment:
You already have a log for the event bellow in switch/case

Reply:
This has changed too. I handle the event twice on porpouse, so in the top of the waterfall you will see the naive/simple implementation for Toasts, and the more complex (where we are aware of specific messages.